updatecli/values.temurin.yaml

```
@@ -0,0 +1,2 @@
+temurin:
+  version_pattern: "^jdk-[11|17|21].(\\d*).(\\d*).(\\d*)(.(\\d*))\\+(\\d*)?$"
```

```suggestion
  version_pattern: "^jdk-[11|17|21].(\\d*).(\\d*).(\\d*)(.(\\d*))\\+(\\d*)?$"

```

(nit) [POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html) says we must have an empty line at the end of a file.

---

I had to merge with `master` anyway, so I updated this file.